README.md

```diff
@@ -0,0 +1,72 @@
+# My INF 7420 Project
+I am going to build a site to support a youth literacy initiative project that a colleague and I are currently working on. 
+
+## Wireframes
+Here are the wireframes for the projects home, about, resources, contact and social media pages. 
+
+### Home Page
+
+![wireframe of Home Page](WIREFRAMES/Home.jpg)
```

Comment:
The case of your image location matters. Please change `WIREFRAMES` to `wireframes` to match the folder/directory your files are in.